src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/locator.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'locator(selector)'
+excerpt: 'Browser module: page.locator(selector) method'
+---
+
+The method returns an element [Locator](/javascript-api/k6-experimental/browser/locator/). Locators resolve to the element when the action takes place, which means locators can span over navigations where the underlying dom changes.
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5